sbc_compassion/data/correspondence_template_data.xml

```diff
@@ -5,6 +5,43 @@
       <field name="name">Default Template</field>
       <field name="layout">CH-A-6S01-1</field>
       <field name="checkbox_ids" eval="False"/>
-    </record>    
+    </record>
+
+    <record id="threshold_keypoints_template" model="ir.config_parameter">
+      <field name="key">threshold_keypoints_template</field>
+      <field name="value">0.25</field>
+    </record>
+
```

Comment:
Above, there is the new parameter for the configuration parameters and below are some parameter moved from demo/